### DISCUSSION TITLE
ERROR: Failed to load resource: the server responded with a status of 404 (Not Found)

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#129] [Help Me!] [answered]
Ellie Walker

Have you searched the other help tickets? 
yes

If there are similar issues, make sure to give us their number/link to them and tell how yours is different.
N/A

Describe your issue. 
I am getting an error when my bear cards are created after clicking the submit button on the form.

List any error messages that appear. 
Failed to load resource: the server responded with a status of 404 (Not Found)

What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
1. https://community.progress.com/s/article/Error-in-console-Failed-to-load-resource-the-server-responded-with-a-status-of-404
2. https://forums.asp.net/t/2120042.aspx?Failed+to+load+resource+the+server+responded+with+a+status+of+404+Not+Found+User+js
3. https://discourse.processing.org/t/facing-error-failed-to-load-resource-the-server-responded-with-a-status-of-404-not-found-in-web-browse-console/2433

What have you tried? Tell us everything you have tried. 
I tried using a different image url when creating a card.
I tried stopping npm and re-starting

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰.
https://github.com/elliewalkersm/ASSIGNMENT-Bear-Watcher/blob/bearForm/src/javascripts/main.js

**Comment by elliewalkersm** [comment, marked as answer]
I changed my for loop to a for each loop and it worked.

**Comment by drteresavasquez** [comment]
This also happened because the image links were not correct, so make sure that when you use links that they are formatted correctly.

**Reply by nashvegasnate** [reply]
I am in the same boat, however, I already had a forEach loop to create my bear card. None of the images will populate and I'm not sure if I have an actual error or if it is something simple like incorrect formatting.
https://github.com/nashvegasnate/ASSIGNMENT-Bear-Watcher/blob/reconstruct/src/javascripts/main.js

**Reply by nashvegasnate** [reply]
Actually...I just figured it out. I had extraneous parentheses and single quotation marks surrounding my ${item.imageUrl} within my function. Problem solved!
